modules/docs/lib/ExampleCodeBlock.tsx

```diff
@@ -70,31 +82,56 @@ export const ExampleCodeBlock = ({code}: any) => {
     navigator.clipboard.writeText(textInput.current.textContent);
   };
 
-  // React.useEffect(() => {
-  //   if (copied) {
-  //     const copyCodeTimeout = setTimeout(() => {
-  //       setCopied(false);
-  //     }, 2000);
+  /**
+   * `code` returns our examples. We need to rewrite them so that they export `Demo`.
+   */
+  const handleExampleRewrite = (code: any) => {
+    return code.replace(/\bexport\s+const\s+(\w+)\s*=/, `export const Demo =`);
```

Comment:
This code will not handle examples that use default exports, but I wasn't able to find any default export examples. We've been removing default exports altogether.